platforms/r3-corda-ent/configuration/README.md

```diff
@@ -1,64 +1,55 @@
 
-## Prerequisites
+## Installation pre-requisites
+BAF requires tools such as Kubernetes, Git (repository), Vault and more to be installed.
+For more information on the installation pre-requisites, please refer to [this guide](https://blockchain-automation-framework.readthedocs.io/en/latest/prerequisites.html).
 
-To run the playbooks, following are the pre-requisites.
-1. Ansible 2.8.2 with jmespath installed (a docker image can be made from this [Dockerfile](../../shared/images/ansibleSlave.Dockerfile)).
-2. Ansible controller configured like this [sample](../../shared/inventory) inventory settings.
-3. One Managed Kubernetes cluster for each organization (currently EKS tested so you need **AWS cli** credentials).
-4. A Hashicorp Vault installation for each organization which is initiated and unsealed. The Vault IP Address should be accessible from this machine (where the playbook is run), and the Vault root token is available.
-5. A Git User with write access to all the branches in the chosen Git repository. This user should be a service user and not a federated user.
-6. Edited and saved network.yaml with all fields populated according to your requirements. A Sample network.yaml can be found [here](./samples/network-cordav2.yaml)
+## Configuration pre-requisites
+For each organization in the DLT network you need to set up the following:
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. Sample `network.yaml`s for R3 Corda Enterprise can be found in [this folder](./samples/).
 
-## Execution 
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
+For other general pre-quisites, such as Docker images, Ambassador and DNS setup, please refer to the ['Configure Pre-requisites' guide](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html).
 
+## Execution 
 ### Step 1
-Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments and readme.
+Ensure that the `network.yaml` is edited properly and saved. For now, follow the guidance in the comment of the sample `network.yaml`(s); a comprehensive guide is WIP.
```

Comment:
Is not WIP

Reply:
True, wording mistake. Resolved in [a9c1489](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/a9c14893b67ea6f9eb1ef2e8a011949764891e53)

Reply:
All corda enterprise details are there in the existing cordanetworkyaml in docs. So give the same link as done for corda opensource.

Reply:
You're correct. The Corda OS documentation has extra distinction for Enterprise. I'll link to this. For future, should we create a seperate page for Corda Enterprise?

Reply:
No, they are the same platform, so no separate operational pages should be created.

Reply:
Resolved in [f95b02a](https://github.com/hyperledger-labs/blockchain-automation-framework/commit/f95b02a3224e22529f3b2556af2b8962c68d78d9)